src/include/err.h

```diff
@@ -17,12 +17,12 @@ static inline void *ERR_PTR(long error)
 }
 */
 
-static inline long PTR_ERR(const void *ptr)
+static inline intptr_t PTR_ERR(const void *ptr)
 {
-  return (uintptr_t) ptr;
+  return (intptr_t) ptr;
 }
 
-static inline long IS_ERR(const void *ptr)
+static inline bool IS_ERR(const void *ptr)
```

Comment:
Need to include `stdbool.h` as well, since it is compiled with a C compiler as well:
```
clang-14: /home/jenkins-build/build/workspace/ceph-pull-requests/build/src/pybind/rados/rados.c
In file included from /home/jenkins-build/build/workspace/ceph-pull-requests/build/src/pybind/rados/rados.c:764:
/home/jenkins-build/build/workspace/ceph-pull-requests/src/include/err.h:25:15: error: unknown type name 'bool'
static inline bool IS_ERR(const void *ptr)
```

Reply:
thanks, I checked with a simple target and didn't notice.